### DISCUSSION TITLE
动漫之家似乎迁移到了再漫画

### DISCUSSION THREAD

**Discussion by lersmare** [xiaoyaocz/flutter_dmzj#225] [General]
动漫之家：https://www.idmzj.com/
再漫画：https://www.zaimanhua.com/

**Comment by zhongfly** [comment]
由于不发工资，动漫之家的编辑/上传全都走了，他们自己另外开了新站：再漫画。

![image](https://github.com/user-attachments/assets/356b87d0-92d7-41e3-8b18-19b7efbebc3a)

**Comment by xiaoyaocz** [comment]
懒得在一个APP里做兼容了，直接开了新分支，用再漫画的数据源搞了个新的APP
https://github.com/xiaoyaocz/flutter_dmzj/releases

**Reply by lersmare** [reply]
感谢ヾ(≧∇≦)
